packages/flame/test/extensions/aabb_test.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:math';
+
+import 'package:flame/extensions.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('aabb extensions tests', () {
+    // aabb2 is an axix aligned bouding box between a min and a max
+    // see https://api.flutter.dev/flutter/vector_math_64/Aabb2-class.html
+    // The extenion is used to convert this bouding box to a rect
+    test('Default aabb constructor', () {
+      final aab2 = Aabb2();
+      // With this constructor, min and max are set to the origin (0,0)
+      // So the rect corresponding should be (0,0,0,0)
+      final aab2Rect = aab2.toRect();
+      _checkRectValues(
+        aab2Rect,
+        left: 0,
+        top: 0,
+        right: 0,
+        bottom: 0,
+      );
+    });
+
+    testRandom('centerAndHalfExtents constructor', (Random r) {
+      // This constructor is usefull in circles (see lib/src/experimental/geometry/shapes/circle.dart)
+      final center = Vector2(r.nextDouble(), r.nextDouble());
+      final halfExtends = Vector2(r.nextDouble(), r.nextDouble());
+
+      final aab2 = Aabb2.centerAndHalfExtents(center, halfExtends);
+      final aab2Rect = aab2.toRect();
+
+      _checkRectValues(
+        aab2Rect,
+        left: aab2.min.x,
+        top: aab2.min.y,
+        right: aab2.max.x,
+        bottom: aab2.max.y,
+      );
+    });
+
+    testRandom('aabb minMax constructor', (Random r) {
+      final min = Vector2(r.nextDouble(), r.nextDouble());
+      final max = Vector2(r.nextDouble(), r.nextDouble());
+
+      final aab2 = Aabb2.minMax(min, max);
+      final aab2Rect = aab2.toRect();
+
+      _checkRectValues(
+        aab2Rect,
+        left: min.x,
+        top: min.y,
+        right: max.x,
+        bottom: max.y,
+      );
+    });
+  });
+}
+
+void _checkRectValues(
+  Rect rect, {
+  // top left angle x
+  required double left,
+  // top left angle y
+  required double top,
+  // bottom right angle's x
+  required double right,
+  // bottom right angle's y
+  required double bottom,
+}) {
```

Comment:
```suggestion
void _checkRectValues(
  Rect rect, {
  required double left,
  required double top,
  required double right,
  required double bottom,
}) {
```

No comments are needed here, the names are self explanatory (and the comments were also wrong since it's not about the angle 😅)